lib/actions.js

```diff
@@ -84,8 +84,15 @@ exports.ensureEmptyDirectory = function() {
   if(fs.existsSync(this.destinationRoot())) {
     // A valid directory won't have any files/folders.
     var files = fs.readdirSync(this.destinationRoot());
+    var file;
     if (files.length > 0) {
-      this.env.error(chalk.red(this.destinationRoot(), 'is not an empty directory'));
+      var count;
```

Comment:
So in the generated SDK use case, a user may want to create a generated server SDK from a swagger file. Since we want to incorporate this SDK into our generated server, it needs to be created before we build our `Package.swift` for example. Thus, at this point in the generator, we may have some iOS or server SDK zip files or folders in an otherwise empty directory. This `ensureEmptyDirectory` was blocking me because of those files, so I added an exception.